inc/Verifiable.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace AntispamBee\Rules;
+
+interface Verifiable {
+	public static function init();
```

Comment:
Should this be part of the interface, or should we allow developers to initialize their filter differently? Would they be able to use our trait as well inside their own classes?